src/AppInstallerCommonCore/Runtime.cpp

```diff
@@ -704,6 +707,21 @@ namespace AppInstaller::Runtime
         return wil::test_token_membership(nullptr, SECURITY_NT_AUTHORITY, SECURITY_BUILTIN_DOMAIN_RID, DOMAIN_ALIAS_RID_ADMINS);
     }
 
+    // Determines whether developer mode is enabled.
+    bool IsDevModeEnabled()
+    {
+        const auto& devModeSubKey = Registry::Key::OpenIfExists(HKEY_LOCAL_MACHINE, s_DevModeSubkey, 0, KEY_READ|KEY_WOW64_64KEY);

```

Comment:
There is also a group policy value that controls it and takes precedence over the user one.  For this temporary use case this is good enough, but might be worth commenting on it.

Reply:
Added a comment clarifying the group policy behavior effect on developer mode setting